### ISSUE TITLE
Improving UI of explore page

### ISSUE THREAD

**Issue by devu007** [Omkar-Sonawane-23/Tourism#264]
**Is your feature request related to a problem? Please describe.**
We can improve the UI of the explore page as it is quite plain now.

**Describe the solution you'd like**
I would like to add the images of places in the background and they will appear like a slide show then.

**Describe alternatives you've considered**

**Additional context**
<img width="1434" alt="image" src="https://github.com/Omkar-Sonawane-23/Tourism/assets/101053111/054f5981-9ef6-4679-a849-390542c09774">

**Comment by devu007** [comment]
Please assign this issue to me under GSSoC 24'

**Comment by sameer-soni** [comment]
@devu007 It's already has been improved, take a loot at it. Would you still like to work on it's improvement?

**Comment by devu007** [comment]
Yes I would like to improve the select box using antd library. By using Card, Select functionalities I can improve it.
